docs/FAQ.md

```diff
@@ -1,3 +1,8 @@
+--
+id: faq.md
+title: FAQ
+--
```

Comment:
Most of these have three dashes, but this one has two and another place has four.  Should they all be three?

Reply:
ah yes, they should, thanks